src/common/wrap/wrap-events.js

```diff
@@ -79,7 +76,32 @@ export function wrapEvents(sharedEE) {
 
   return ee
 }
+/**
+ * Find the base prototype of 'object' that has its own "addEventListener" property, and run some function on it.
+ * @param {Object} object - the initial object to traverse prototype chain on
+ * @param {Function} cb - the function to run on the ancestral object once found, accepts an object as a arg
+ * @param {Array} rest - [optional] any additional arguments to pass to the cb
+ */
+function findEventListenerProtoAndCb(object, cb, ...rest) {
+  let step = object;
+  // eslint-disable-next-line
+  while (step && !step.hasOwnProperty(ADD_EVENT_LISTENER)) { step = Object.getPrototypeOf(step) }
+  if (step) cb(step, ...rest);
+}
 
+export function unwrapEvents(sharedEE) {
+  const ee = scopedEE(sharedEE);
+  
+  // Don't unwrap until the LAST of all features that's using this (wrapped count) no longer needs this, but always decrement the count after checking it per unwrap call.
+  if (wrapped[ee.debugId]-- == 1) {
```

Comment:
This can be confusing and I would suggest we don't decrement the variable inside the logic check.

Reply:
Moved into `else` case.